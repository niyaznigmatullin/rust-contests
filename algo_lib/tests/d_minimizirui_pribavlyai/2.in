7
1 7 2 4 8 4 100
10
1 3 6 3
3 2 7
1 2 3 5
2 3 4 -10
3 1 7
1 1 7 3
3 1 4
3 2 7
2 1 7 5
3 1 7
