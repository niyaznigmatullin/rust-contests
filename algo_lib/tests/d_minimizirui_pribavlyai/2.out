118
97
-11
-3
33
